cmd/goal/account.go

```diff
@@ -1020,3 +1028,45 @@ var partkeyInfoCmd = &cobra.Command{
 		})
 	},
 }
+
+var markNonparticipatingCmd = &cobra.Command{
+	Use:   "marknonparticipating",
+	Short: "Mark an account as not participating",
```

Comment:
Kind of a nitpick, but could we add something like "(i.e., offline and earns no rewards)" to the short description? Otherwise I could imagine people seeing this and marking themselves nonparticipating when they really mean to mark themselves offline.